connPFM/debiasing/debiasing.py

```diff
@@ -1,31 +1,31 @@
 """Main debiasing workflow."""
 import logging
-import subprocess
 from os.path import join
 
 import numpy as np
-from nilearn.input_data import NiftiLabelsMasker
 
 from connPFM.debiasing.debiasing_functions import debiasing_spike  # or debiasing_block
-from connPFM.utils import atlas_mod
+from connPFM.utils import io
 from connPFM.utils.hrf_generator import HRFMatrix
 
 LGR = logging.getLogger(__name__)
 
 
-def debiasing(data_file, mask, mtx, tr, out_dir, prefix, groups, groups_dist, history_str):
+def debiasing(data_file, mask, te, mtx, tr, out_dir, prefix, groups, groups_dist, history_str):
     """Perform debiasing based on denoised edge-time matrix."""
-    LGR.info("Performing debiasing based on denoised edge-time matrix...")
-    masker = NiftiLabelsMasker(
-        labels_img=mask,
-        standardize=False,
-        strategy="mean",
-    )
+    if te is None:
```

Comment:
```suggestion
    if te is None and len(data_file == 1):
```

Reply:
I like this idea. I'm adding it to all 3 workflows.